src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$gray: #eee;
+$black: #000;
+$pink: #ffbfcb;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$day-block-length: 100px;
```

Comment:
```suggestion
$day-size: 100px;
```

Reply:
Yes, I agree that name is a bit wired. Next time I'll think of a better name for variables.